nengo/tests/test_neurons.py

```diff
@@ -281,7 +281,7 @@ def test_dt_dependence(Simulator, nl_nodirect, plt, seed, rng):
     """Neurons should not wildly change with different dt."""
     with nengo.Network(seed=seed) as m:
         m.config[nengo.Ensemble].neuron_type = nl_nodirect()
-        u = nengo.Node(output=WhiteSignal(0.1, 5), size_out=2)
+        u = nengo.Node(output=WhiteSignal(0.1, high=8), size_out=2)
```

Comment:
Also, when I fix this, this test breaks because the WhiteSignal changes depending on the `dt` used. In general, I think it would be very difficult to have our random processes be agnostic to dt. But do you think users will expect this? In this respect, our old method of generating WhiteSignal was better, where we had sine waves at fixed frequency intervals and we just generated random coefficients for them, since that method is agnostic to `dt`.